automation/jenkins_build.sh

```diff
@@ -35,17 +75,22 @@ if [ -z "$BUILD_NUMBER" ] || [ -z "$WORKSPACE" ] || [ -z "$sourceBranch" ] || [
     exit 1
 fi
 
-# Development images based on environment variable
-if [ -n "$DEVELOPMENT_IMAGE" ]; then
-    echo "[INFO] Running a development build..."
-    BARYS_ARGUMENTS_VAR="$BARYS_ARGUMENTS_VAR --development-image"
-fi
-
-if [ "z$BUILD_TYPE" == "zresinos" ]; then
-    echo "[INFO] Running a resinOS build..."
-else
-    echo "[INFO] Running a resinIO build..."
-    BARYS_ARGUMENTS_VAR="$BARYS_ARGUMENTS_VAR --resinio"
+if [ "$buildFlavor" == "managed-dev" ]; then
+	BARYS_ARGUMENTS_VAR="$BARYS_ARGUMENTS_VAR --resinio"
+	BARYS_ARGUMENTS_VAR="$BARYS_ARGUMENTS_VAR --development-image"
+	DEVELOPMENT_IMAGE=yes
+	RESIN_MANAGED_IMAGE=yes
+elif [ "$buildFlavor" == "managed-prod" ]; then
+	BARYS_ARGUMENTS_VAR="$BARYS_ARGUMENTS_VAR --resinio"
+	DEVELOPMENT_IMAGE=no
+	RESIN_MANAGED_IMAGE=yes
+elif [ "$buildFlavor" == "unmanaged-dev" ]; then
+	BARYS_ARGUMENTS_VAR="$BARYS_ARGUMENTS_VAR --development-image"
+	DEVELOPMENT_IMAGE=yes
+	RESIN_MANAGED_IMAGE=no
+elif [ "$buildFlavor" == "unmanaged-prod" ]; then
+	DEVELOPMENT_IMAGE=no
+	RESIN_MANAGED_IMAGE=no
```

Comment:
It might be better to consolidate making the argument list from the DEVELOPMENT_IMAGE and RESIN_MANAGED_IMAGE flags, e.g.:

    if [ $DEVELOPMENT_IMAGE = "yes" ] ; then
        BARYS_ARGUMENTS_VAR="$BARYS_ARGUMENTS_VAR --development-image"
    fi
    ...